src/server/search/search_family_test.cc

```diff
@@ -757,4 +757,42 @@ TEST_F(SearchFamilyTest, Vector) {
   EXPECT_EQ(resp, "OK");
 }
 
+TEST_F(SearchFamilyTest, EscapedSymbols) {
+  Run({"ft.create", "i1", "ON", "HASH", "SCHEMA", "color", "tag"});
+
+  // TODO ',' is separator, we need to check should next request work or not
```

Comment:
can you elaborate on this? what do you mean that ',' is a separator?

Reply:
',' is used as a default (we can change the separator)  separator in the tag list and when it is used as a separator we can not find the whole tag that contains it. I'm not sure how it should work, because even in Redis it works for JSON but not for HASH

Reply:
I am afraid we reach the point where we start mirroring Redis bugs. I prefer to be as restrictive as possible. So if quoted string 'foo,bar' as a tag work in dragonjfly I would like to remove `,` from `tag_val_char`

Reply:
If the separator will be changed to '-' it will be the same issue but with '-'